server/plugin_test.go

```diff
@@ -339,7 +339,7 @@ func TestSyncUsers(t *testing.T) {
 		{
 			Name: "SyncUsers: Unable to create the user",
 			SetupAPI: func(api *plugintest.API) {
-				api.On("LogError", "Unable to create new MM user during sync job", "email", "test@test.com", "error", mock.Anything).Times(1)
+				api.On("LogError", "Unable to create new MM user during sync job", "MMUserID", mock.Anything, "TeamsUserID", mock.Anything, "error", mock.Anything).Times(1)
```

Comment:
Note that these API assertions don't actually test anything since we never call `api.AssertExpectations(t)`, but updated for completeness. (I can't add `api.AssertExpectations(t)` without failing all the tests. Ideally, we transition these away from mocks altogether.)